index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-us">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Zema's Portfolio</title>
   <link rel="stylesheet" href="styles.css">
-  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Hello Microverse!</h1>
+  <header>
+    <div class="status-bars">
+
+    </div>
+    <div class="app-bar">
+      <div class="toolbar">
+        <nav>
+          <div class="normal-button">
+            <img class="burger" src="assets/burger.svg" alt="Menu icon.">
+          </div>
+        </nav>
+        <a href="#home">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
```

Comment:
- Please, kindly ensure your logo is wrapped inside the nav bar to meet the project requirement.